### DISCUSSION TITLE
Any Text Installer for Mocaccino?

### DISCUSSION THREAD

**Discussion by DerMonozukurist** [mocaccinoOS/mocaccino#128] [General]
I'm building general purpose linux server for my home. E.g., DNS cache, DHCP. Would be great if there's an installation guide without the use of Calamares, or better yet a text installer.

**Comment by joostruis** [comment]
No. But after install you can remove most layers anyway.
Just keep in mind that NetworkManager is in layers/X and thus you must configure your network differently if you really want to remove that layer. 

https://www.mocaccino.org/docs/general/network-config/#network-manager

**Comment by DerMonozukurist** [comment]
There isn't much documentation about "layers", but I kind of thought it's similar to "kits" in Funtoo. 

The problem with not having a text installer is that if X don't work, then you cannot install Mocaccino. Case in point, mine segfaulted even when using X's FB module, and for some reason I can't use my computer's GPU.

I did however managed to "install" it using `mocaccino-unnatended-installer`. After installation, I uninstalled unneeded package by `luet uninstall layer/X --full-clean`, because `luet uninstall layer/X --full` did not proceed; but only to end up with a non working system.

**Comment by joostruis** [comment]
mocaccino-unnatended-installer is not meant for this.

**Reply by DerMonozukurist** [reply]
given the lack of documentation, could you enlighten me pls about its use?

**Reply by DerMonozukurist** [reply]
It gets the job done, does it not?
